dart/dynamics/Joint.h

```diff
@@ -347,6 +354,16 @@ class Joint : public virtual common::Subject
   /// Get upper limit for position
   virtual double getPositionUpperLimit(size_t _index) const = 0;
 
+  /// Gets whether a generalized coordinate is cyclic. Returns true if and only
```

Comment:
@mxgrey's argument makes sense for documenting functions, like getters and setters, but it doesn't work for other types of documentation. For example, like in @psigen's example, a class may have the comment that says something like "represents a rigid body transformation" or "attaches a `Node` to a `Skeleton`."

It doesn't make sense for the developer to the subject of the sentence in those cases. I suppose we could make a separate rule for function documentation (which it sounds like Python does). Or we could always use `s`.